### ISSUE TITLE
report: The Retargetable Decompiler (part of Related Work) [6h]

### ISSUE THREAD

**Issue by mewmew** [decomp/decomp#47]
Depends on issue #81.

**Comment by mewmew** [comment]
Prerequisite for issue #35.

**Comment by mewmew** [comment]
Merged into the Decompilers section. Closing this issue in favour of the issue of the Decompilers section.
